src/storage/src/client/connections.rs

```diff
@@ -713,3 +729,13 @@ impl Arbitrary for PostgresConnection {
             .boxed()
     }
 }
+
+/// A connection to a SSH tunnel.
+#[derive(Arbitrary, Clone, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
+pub struct SshConnection {
+    pub host: String,
+    pub port: i32,
+    pub user: String,
+    pub public_key: String,
+    pub private_key: String,
```

Comment:
`private_key` should be a secret!

```suggestion
    pub private_key: GlobalId,
```